NightscoutUploadKit/Models/GlucoseEntry.swift

```diff
@@ -119,6 +132,8 @@ public struct GlucoseEntry {
             self.trend = GlucoseTrend(rawValue: intTrend)
         } else if let stringTrend = rawValue["trend"] as? String, let intTrend = Int(stringTrend) {
             self.trend = GlucoseTrend(rawValue: intTrend)
```

Comment:
I don't normally include types in my variable names but did so here to avoid confusion in the parsing of these various values.